www/front_src/src/Header/userMenu/index.tsx

```diff
@@ -0,0 +1,307 @@
+
+const EDIT_PROFILE_TOPOLOGY_PAGE = '50104';
+
+const useStyles = makeStyles(() => ({
+  fullname: {
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    width: '115px',
+  },
+  itemLink: {
+    backgroundColor: '#232f39',
+    color: 'white',
+    display: 'flex',
+    flexDirection: 'row',
+    fontSize: '0.8rem',
+    justifyContent: 'space-between',
+    padding: 10,
+    textDecoration: 'none',
+  },
+  logoutLink: {
+    display: 'grid',
+    justifyContent: 'flex-end',
+  },
+  nameAliasContainer: {
+    display: 'grid',
+    gridTemplateColumns: '2fr 1fr',
+  },
+  subMenu: {
+    boxSizing: 'border-box',
+    display: 'none',
+    height: '100vh',
+    left: 0,
+    overflowY: 'auto',
+    position: 'absolute',
+    textAlign: 'left',
+    top: '100%',
+    width: '100%',
+    zIndex: 92,
+  },
+  subMenuActive: {
+    display: 'block',
+  },
+  subMenuItemContent: {
+    backgroundColor: '#232f39',
+    padding: 10,
+  },
+  submenuUserButton: {
+    '& span:first-child': {
+      display: 'block',
+      lineHeight: '17px',
+    },
+    '&:hover': {
+      backgroundColor: 'rgba(255, 161, 37, 1)',
+      color: 'rgb(0,9,22)',
+    },
+    alignItems: 'center',
+    backgroundColor: 'transparent',
+    border: '1px solid rgba(255, 161, 37, 1)',
+    borderRadius: '16px',
+    boxSizing: 'border-box',
+    color: '#ffa225',
+    display: 'flex',
+    fontSize: '.7rem',
+    justifyContent: 'space-between',
+    lineHeight: '31px',
+    margin: '0 auto',
+    outline: 'none',
+    padding: '2px 10px 2px 10px',
+    position: 'relative',
+    textAlign: 'left',
+    textDecoration: 'none',
+    width: '95%',
+  },
+  submenuUserEdit: {
+    color: 'white',
+    float: 'right',
+    marginLeft: 4,
+    textDecorationLine: 'underline',
+  },
+  wrapRightUser: {
+    alignItems: 'center',
+    display: 'flex',
+    flexWrap: 'wrap',
+    padding: '6px 22px 6px 61px',
+    position: 'relative',
+  },
+  wrapRightUserActive: {
+    backgroundColor: '#000915',
+  },
+  wrapRightUserItems: {
+    display: 'flex',
+    flex: '1 0 76%',
+    justifyContent: 'flex-end',
+  },
+}));
+
+// useRef
+let refreshTimeout;
+
+const UserMenu = ({ allowedPages }: StateToProps): JSX.Element => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  const [copied, setCopied] = React.useState(false);
+  const [data, setData] = React.useState<any>(null);
+  const [toggled, setToggled] = React.useState(false);
+  const profile = React.useRef<any>();
+  const autologinNode = React.useRef<any>();
+
+  React.useEffect(() => {
+    window.addEventListener('mousedown', handleClick, false);
+    getData();
+
+    return (): void => {
+      window.removeEventListener('mousedown', handleClick, false);
+      clearTimeout(refreshTimeout);
+    };
+  }, []);
+
+  // fixer par useRequest pollerMenu
+  const userService = axios(
+    'internal.php?object=centreon_topcounter&action=user',
+  );
```

Comment:
We have a custom hook `useRequest` in order to send HTTP request (Example: https://github.com/centreon/centreon/blob/develop/www/front_src/src/Header/PollerMenu/ExportConfiguration/index.tsx#L51)